Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs

```diff
@@ -225,7 +224,14 @@ public override void Execute()
             }
             else if (SetVariableValueOption == VariableBase.eSetValueOptions.AutoGenerateValue)
             {
-                ((VariableBase)Var).GenerateAutoValue();
+                isAutoGenerateValuesucceed = ((VariableBase)Var).GenerateAutoValue(ref errorMsg);
```

Comment:
1- write it in single line:
if (!(VariableBase)Var).GenerateAutoValue(ref errorMsg))
{
 Error = errorMsg;
}

you should not have empty else block